### PR TITLE
Only set scanout flag for display buffer

### PR DESCRIPTION
**Pull request by yhe39**

### CODE REVIEW

File: cros_gralloc/cros_gralloc_helpers.cc [hunk 1]
```diff
@@ -26,6 +26,9 @@
 /* Define to match AIDL PixelFormat::R_8. */
 #define HAL_PIXEL_FORMAT_R8 0x38
 
+/* Define to match BufferUsage::GRALLOC_USAGE_PRIVATE_1. */
+#define GRALLOC_USAGE_PRIVATE_1 (1 << 29)
+
```
Comment (chenyanxzhu) [inline cros_gralloc/cros_gralloc_helpers.cc]: This is already defined as enum in "hardware/libhardware/include/hardware/gralloc.h", which is included in this file already. So maybe remove these 2 lines.

File: cros_gralloc/cros_gralloc_helpers.cc [hunk 2]
```diff
@@ -26,6 +26,9 @@
 /* Define to match AIDL PixelFormat::R_8. */
 #define HAL_PIXEL_FORMAT_R8 0x38
 
+/* Define to match BufferUsage::GRALLOC_USAGE_PRIVATE_1. */
+#define GRALLOC_USAGE_PRIVATE_1 (1 << 29)
```
Comment (phreer) [inline cros_gralloc/cros_gralloc_helpers.cc]: Could you elaborate on the sematics of this flag? And maybe a more descriptive name for this flag would be better?

File: cros_gralloc/cros_gralloc_helpers.cc [hunk 3]
```diff
@@ -138,8 +141,8 @@ uint64_t cros_gralloc_convert_usage(uint64_t usage)
 	handle_usage(&usage, GRALLOC_USAGE_HW_RENDER, &use_flags, BO_USE_RENDERING);
 	handle_usage(&usage, GRALLOC_USAGE_HW_2D, &use_flags, BO_USE_RENDERING);
 	/* HWC wants to use display hardware, but can defer to OpenGL. */
-	handle_usage(&usage, GRALLOC_USAGE_HW_COMPOSER, &use_flags,
-		     BO_USE_SCANOUT | BO_USE_TEXTURE);
+	handle_usage(&usage, GRALLOC_USAGE_HW_COMPOSER, &use_flags, BO_USE_TEXTURE);
+	handle_usage(&usage, GRALLOC_USAGE_PRIVATE_1, &use_flags, BO_USE_SCANOUT);
```
Comment (phreer) [inline cros_gralloc/cros_gralloc_helpers.cc:142]: I haven't a clue about why what is happening here and this change is required, so please provide some background.

**Review by feijiang1 — APPROVED** [review]
LGTM

**Review by phreer — APPROVED** [review]

**Review by chenyanxzhu — APPROVED** [review]

**Review by feijiang1 — APPROVED** [review]
LGTM

**Pull request merged**